core/src/main.cpp

```diff
@@ -847,6 +860,37 @@ auto load_oidc_configuration(const json& _config, json& _oi_config, json& _endpo
 	return true;
 } // load_oidc_configuration
 
+auto load_user_mapping_plugin(const json& _oi_config) -> bool {
+	const auto map_lib_iter{_oi_config.find("user_mapping_shared_library")};
+	const auto config_string_iter{_oi_config.find("user_mapping_configuration")};
+
+	if (const auto config_end{std::end(_oi_config)}; map_lib_iter == config_end || config_string_iter == config_end) {
+		logging::warn("[{}]: missing 'user_mapping_shared_library' (and/or) 'user_mapping_configuration'.", __func__);
+		return false;
+	}
```

Comment:
Consider moving this check outside the function, similar to the `load_oidc_configuration(...)` function.

Reply:
Related: https://github.com/irods/irods_client_http_api/pull/331#discussion_r1747231960

Reply:
Unneeded.